src/test/scala/cognite/spark/v1/RawTableRelationTest.scala

```diff
@@ -457,8 +482,40 @@ class RawTableRelationTest
       try {
         writeClient.rawTables(database).deleteById(table)
       } catch {
-        case _: CdpApiException => ()// Ignore
+        case _: CdpApiException => () // Ignore
       }
     }
   }
+
+  // This is a test for a Temp fix applied
+  it should "not allow to fetch a large number of columns which has 200 characters when combined with ','" taggedAs WriteTest in {
```

Comment:
The test is good! I suggest rewriting it to somehow check that all rows are _fetched_.. it's ok to just check that the read actually works though, since we know it'll be hitting the right codepath if it reads the data instead of getting an exception from RAW.

Would actually be even better if you 
```
val columnsAsString = columns.mkString(",")
s"select $columnsAsString from rw.tbl"
```